### PR TITLE
Fix/country selector

### LINKED ISSUE #98: [PROFILES] Have a working Country Selector
## Description
The country selector should work as expected - when a user selects a country, they are taken to the right profile and if they are looking at a country profile (or its sections), the country selector will be showing the right one.

## Screenshots

screen shot 2018-06-19 at 10 44 04

## Tasks

- [x] Determine the profile being viewed, have that country's flag and name as the button's content.

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

The country selector should work as expected - when a user selects a country, they are taken to the right profile and if they are looking at a country profile (or its sections), the country selector will be showing the right one.

Fixes #98 
screen shot 2018-06-29 at 11 17 44

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
screen shot 2018-06-29 at 11 20 25

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/takwimu/_includes/profile/country_selector.html [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
-        border="8" style="border-style:double; width: 50px; height: 50px;"/>
-    Nigeria
+
+  <button class="btn btn-outline-secondary btn-lg dropdown-toggle" type="button"
+          id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true"
+          aria-expanded="false">
+    <img class="rounded-circle border-light mr-2"
+         src={% with 'img/flags/'|add:current_page.title|lower|add:'.svg' as image_static %}
+           {% static image_static %}
+         {% endwith %}
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/profile/country_selector.html]: Aren't quotes needed i.e `src="..."`?

File: takwimu/templates/takwimu/_includes/profile/country_selector.html [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
+
+    {% for profile in current_page.get_siblings %}
+      <a class="dropdown-item" href="{{ profile.get_url }}"
+         data-value="{{ profile.title | lower }}"><img
+          class="rounded-circle border-light mr-2"
+          src={% with 'img/flags/'|add:profile.title|lower|add:'.svg' as image_static %}
+            {% static image_static %}
+          {% endwith %} border="8"
+          style="border-style:double; width: 50px; height: 50px;"/>{{ profile.title }}
+      </a>
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/profile/country_selector.html]: Same here. Aren't quotes needed i.e `src="..."`?

Comment (phillipahereza) [reply takwimu/templates/takwimu/_includes/profile/country_selector.html]: @DavidLemayian the static tag adds them automatically quotes, so adding them is redundant

Comment (DavidLemayian) [reply takwimu/templates/takwimu/_includes/profile/country_selector.html]: I stand corrected. All the examples show quotes but I suppose this is a force of habit.

**Pull request merged**